src/main/scala/mesosphere/marathon/core/storage/store/impl/zk/RichCuratorFramework.scala

```diff
@@ -147,12 +147,12 @@ class RichCuratorFramework(val client: CuratorFramework) {
     * @param lifecycleState reference to interface to query Marathon's lifecycle state
     */
   @SuppressWarnings(Array("CatchFatal"))
-  def blockUntilConnected(lifecycleState: LifecycleState): Unit = {
+  def blockUntilConnected(lifecycleState: LifecycleState, crashStrategy: CrashStrategy): Unit = {
     if (!lifecycleState.isRunning)
       throw new InterruptedException("Not waiting for connection to zookeeper; Marathon is shutting down")
 
-    if (!client.blockUntilConnected(client.getZookeeperClient.getConnectionTimeoutMs, java.util.concurrent.TimeUnit.MILLISECONDS))
-      throw new InterruptedException("Timed out while waiting for zookeeper connection")
+    if (!client.blockUntilConnected(100, java.util.concurrent.TimeUnit.MILLISECONDS))
```

Comment:
What is the motivation to hardcode 100 instead of using `client.getZookeeperClient.getConnectionTimeoutMs`?

Reply:
I was just testing if you're paying attention - you passed with flying colors! 🎉